src/pipeline/base_pipeline.py

```diff
@@ -5,6 +5,7 @@
 from abc import ABC, abstractmethod
 
 import torch
+import pandas as pd
```

Comment:
pandas is not in env config

Reply:
(perhaps update the environment.yaml)

Reply:
Actually the following errors start to pop out when we don't update `environment.yml` for some time.
![image](https://user-images.githubusercontent.com/17662095/72135221-e95ac980-33c0-11ea-8ac6-f433d8068166.png)

Reply:
Maybe a better way is to provide a script like this?
```
conda create -n PytorchGoldenTemplate python=3.7 pytorch=1.3.0 pandas ...
source activate PytorchGoldenTemplate
pip install attrs entrypoints ...
```

Reply:
Just like [what I just added](https://github.com/amjltc295/pytorch-golden-template/pull/75/commits/954710df214c380a230f4685eede15c737f1ccd6)

Reply:
I put it to readme.md and updated the environment.yaml as well